splinterd/src/config/toml.rs

```diff
@@ -37,7 +37,7 @@ struct TomlConfig {
     peers: Option<Vec<String>>,
     node_id: Option<String>,
     display_name: Option<String>,
-    bind: Option<String>,
+    rest_api_endpoint: Option<String>,
```

Comment:
Bind should be added to the list of deprecated values below

Reply:
Added